src/main/java/ro/societateahermes/backendservice/controller/EventController.java

```diff
@@ -0,0 +1,25 @@
+package ro.societateahermes.backendservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import ro.societateahermes.backendservice.service.EventServiceInterface;
+import ro.societateahermes.backendservice.service.dto.NotificationSwitchDTO;
+
+@RestController
+public class EventController {
```

Comment:
Poti si e recomandat sa declari un path general pentru controller. Poti scrie @RequestMapping("event") in cazul asta deasupra clasei